lib/deploy/events/apiGateway/apigateway-to-stepfunctions-assume-role.json

```diff
@@ -22,7 +22,8 @@
             {
               "Effect": "Allow",
               "Action": [
-                "states:StartExecution"
+                "states:StartExecution",
+                "states:DescribeExecution"
```

Comment:
maybe take this permission-generation logic out, and put it into the code so that we only give the necessary permissions if the user configures an endpoint to describe execution